.github/workflows/main.yml

```
@@ -42,8 +42,12 @@ jobs:
           bundler-cache: true
       - name: Decode and Write Google Service JSON
         env:
-            GOOGLE_SERVICES_JSON: ${{ secrets.GOOGLE_SERVICES_JSON }}
+          GOOGLE_SERVICES_JSON: ${{ secrets.GOOGLE_SERVICES_JSON }}
         run: echo $GOOGLE_SERVICES_JSON | base64 --decode >./app/google-services.json
+      - name: Decode and Write Secrets
+        env:
+          SERVER_CLIENT_ID: ${{ secrets.SERVER_CLIENT_ID }}
+        run: echo $SERVER_CLIENT_ID | base64 --decode >./modules/account/src/main/res/values/secrets.xml
```

We might want to rename this to `SECRETS_XML`
```suggestion
      - name: Decode and Write Secrets
        env:
          SECRETS_XML: ${{ secrets. SECRETS_XML }}
        run: echo $SECRETS_XML | base64 --decode >./modules/account/src/main/res/values/secrets.xml
```